ethereum/core/src/test/java/tech/pegasys/pantheon/ethereum/permissioning/PermissioningConfigurationTest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2018 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.ethereum.permissioning;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Arrays;
+
+import org.junit.Test;
+
+public class PermissioningConfigurationTest {
+
+  @Test
+  public void defaultConfiguration() {
+    final PermissioningConfiguration configuration = PermissioningConfiguration.createDefault();
+    assertThat(configuration.getNodeWhitelist()).isEmpty();
+    assertThat(configuration.isNodeWhitelistSet()).isFalse();
+  }
+
+  @Test
+  public void setNodeWhitelist() {
+    final String[] nodes = {"enode://001@123:4567", "enode://002@123:4567", "enode://003@123:4567"};
```

Comment:
Question: should we validate if the enode in the config file is valid? We are planning to validate it in the JSON-RPC API so maybe it makes sense to do the same when reading from the file.
Thoughts?

Reply:
https://pegasys1.atlassian.net/browse/NC-1981